src/libstd/path.rs

```diff
@@ -10,20 +10,20 @@
 
 //! Cross-platform path manipulation.
 //!
-//! This module provides two types, `PathBuf` and `Path` (akin to `String` and
-//! `str`), for working with paths abstractly. These types are thin wrappers
-//! around `OsString` and `OsStr` respectively, meaning that they work directly
+//! This module provides two types, [`PathBuf`] and [`Path`] (akin to [`String`] and
```

Comment:
`[Path] (akin to...)` seems to be interpreted as a Markdown link, causing Travis failure in linkcheck.